components/tools/OmeroWeb/omeroweb/webgateway/views.py

```diff
@@ -2745,7 +2745,8 @@ def _annotations(request, objtype, objid, conn=None, **kwargs):
     links = [l for obj in objs for l in obj.copyAnnotationLinks()]
     for link in links:
         annotation = link.child
-        if unwrap(annotation.getNs()) != NSBULKANNOTATIONS:
+        if (not isinstance(annotation, omero.model.FileAnnotation) or
+                unwrap(annotation.getNs()) != NSBULKANNOTATIONS):
```

Comment:
Re: review of the code. I can review (since these were @manics changes). This is the only critical fix. Everything else is just about docs and testing. And this looks good (and works). 👍